templates/students/attendance.html

```diff
@@ -178,7 +176,12 @@ <h3 class="text-center">Students Attendance</h3>
                 $("#stu-att-table").html(stu_att_table);
             }
         });
-    });
+    }
+    
+    // $("#stu-att-class option:selected").removeAttr('selected');
+    $("#stu-att-class option:first").attr('selected', 'selected');
+    $("#stu-att-class").ready(fetchStudentsForAttendance);
```

Comment:
Need for this? It used to work without `.ready` before, so what changed now?

Reply:
Without `.ready`, the page was loading the data for classes 1-3 only. On digging deeper, I found that on line 200 of `views.py` of students app, we had set the initial data load for classes 1-3 only for initial load. So, instead of changing that, I thought that it would be better to use a ready function, so that we don't have to change that value each time we change the default option in the dropdown.


![image](https://user-images.githubusercontent.com/68158825/194390732-307c2f27-7c44-4d92-9d42-0a71ee5e54ac.png)

Reply:
Cool.